lib/shopify_api/resources/fulfillment_order.rb

```diff
@@ -12,5 +12,41 @@ def fulfillments(options = {})
       fo_fulfillments = get(:fulfillments, options)
       fo_fulfillments.map { |fof| FulfillmentOrderFulfillment.new(fof.as_json) }
     end
+
+    def move(new_location_id:)
+      body = { fulfillment_order: { new_location_id: new_location_id } }.to_json
+      keyed_fos = load_keyed_attributes_from_response(post(:move, {}, body))
+      if keyed_fos&.fetch('original_fulfillment_order', nil)&.attributes
+        load(keyed_fos['original_fulfillment_order'].attributes, false, true)
+      end
+      keyed_fos
+    end
+
+    def cancel
+      keyed_fos = load_keyed_attributes_from_response(post(:cancel, {}, only_id))
+      if keyed_fos&.fetch('fulfillment_order', nil)&.attributes
```

Comment:
same question as above about why specifically this key ?

Reply:
This key represents the mutation of the subject FO resource issuing the `.cancel`.